packages/pharos/src/components/progress-bar/pharos-progress-bar.ts

```diff
@@ -36,18 +59,12 @@ export class PharosProgressBar extends PharosElement {
         aria-valuemax="100"
         aria-valuenow=${this.value}
         aria-labelledby="title"
+        aria-describedby="description"
         class="progress-bar__wrapper"
       >
         <div
-          class="progress-bar"
-          style=${styleMap({
-            background: `linear-gradient(
-              to right,
-              ${PharosColorGlacierBlueBase} ${this.value < 35 ? '100%' : 135 - this.value + '%'},
-              ${PharosColorNightBlueBase}
-            )`,
-            width: `${this.value}%`,
-          })}
+          class="${classList}"
+          style=${progressStyle}
         ></div>
       </div>
       <div class="progress-bar__description">
```

Comment:
```suggestion
      <div class="progress-bar__description" id="description">
```

Ensuring that the newly added `aria-describedby` has a valid ID ref.